apps/nuxt3-ssr/components/FileDocumentCard.vue

```diff
@@ -13,6 +13,7 @@ defineProps({
   <a
     class="border rounded-3px p-5 flex items-center hover:shadow-md transition-shadow"
     :href="url"
+    target="_blank"
```

Comment:
may depend on document type , need to look into details of functional spec. say its an image we may want to show it in a lightbox ( with option to download), if its a 'file' we may want to download directly, if its a link to a other ( external ) page then ref target black. Need to check data in db for options used currently and check schema setting for possible usge

Reply:
Please makes sure issues are defined better in that case. Closing the PR.